src/main/java/appeng/fluids/client/gui/GuiFluidInterface.java

```diff
@@ -0,0 +1,53 @@
+
+package appeng.fluids.client.gui;
+
+
+import net.minecraft.entity.player.InventoryPlayer;
+
+import appeng.client.gui.AEBaseGui;
+import appeng.client.gui.widgets.GuiFluidTank;
+import appeng.core.localization.GuiText;
+import appeng.fluids.container.ContainerFluidInterface;
+import appeng.fluids.helper.DualityFluidInterface;
+import appeng.fluids.helper.IFluidInterfaceHost;
+
+
+public class GuiFluidInterface extends AEBaseGui
+{
+	public final static int ID_BUTTON_TANK = 222;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)